docs/content/common-subs/kubestellar-test-apache.md

```diff
@@ -0,0 +1,62 @@
+<!--kubestellar-test-apache-start-->
+Now, let's check that the deployment was created in the kind **ks-edge-cluster1** cluster (it may take up to 30 seconds to appear):
+```shell
+KUBECONFIG=~/.kube/config kubectl --context ks-edge-cluster1 get deployments -A
+```
+
+you should see output including:
+``` { .sh .no-copy }
+NAMESPACE           NAME                              READY   UP-TO-DATE   AVAILABLE   AGE
+my-namespace        my-first-kubestellar-deployment    1/1        1            1       6m48s
+```
+
+And, check the **ks-edge-cluster2** kind cluster for the same:
+```shell
+KUBECONFIG=~/.kube/config kubectl --context ks-edge-cluster2 get deployments -A
+```
+
+you should see output including:
+``` { .sh .no-copy }
+NAMESPACE           NAME                              READY   UP-TO-DATE   AVAILABLE   AGE
+my-namespace        my-first-kubestellar-deployment    1/1        1            1       7m54s
+```
+
+Finally, let's check that the workload is working in both clusters:
+For **ks-edge-cluster1**:
+```shell
+export KUBECONFIG=~/.kube/config
+while [[ $(kubectl --context ks-edge-cluster1 get pod \
+  -l "app=common" -n my-namespace \
+  -o jsonpath='{.items[0].status.phase}') != "Running" ]]; \
+  do sleep 5; done;curl http://localhost:8094
```

Comment:
Editing oversight here, I assume you did not want to tack the `curl` onto the end of the `while`.